source/setup.py

```diff
@@ -311,6 +311,7 @@ def _genManifestTemplate(shouldHaveUIAccess: bool) -> tuple[int, int, bytes]:
 				"*.xliff",
 				"*/user_docs/styles.css",
 				"*/user_docs/numberedHeadings.css",
+				"*/source/images/nvda.ico",
```

Comment:
We actually want to keep including this

```suggestion
```

Reply:
However, we probably want to remove the installed file

```suggestion
				"*/user_docs/favicon.ico",
```